x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_process_list.tsx

```diff
@@ -1,199 +1,203 @@
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { memo, useContext, useCallback, useMemo } from 'react';
+import {
+  EuiBasicTableColumn,
+  EuiBadge,
+  EuiButtonEmpty,
+  EuiSpacer,
+  EuiInMemoryTable,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import * as event from '../../../../common/endpoint/models/event';
+import * as selectors from '../../store/selectors';
+import { CrumbInfo, formatter, StyledBreadcrumbs } from './panel_content_utilities';
+import { useResolverDispatch } from '../use_resolver_dispatch';
+import { SideEffectContext } from '../side_effect_context';
+import { CubeForProcess } from './process_cube_icon';
+import { SafeResolverEvent } from '../../../../common/endpoint/types';
+import { LimitWarning } from '../limit_warnings';
+
+const StyledLimitWarning = styled(LimitWarning)`
+  flex-flow: row wrap;
+  display: block;
+  align-items: baseline;
+  margin-top: 1em;
+
+  & .euiCallOutHeader {
+    display: inline;
+    margin-right: 0.25em;
+  }
+
+  & .euiText {
+    display: inline;
+  }
+
+  & .euiText p {
+    display: inline;
+  }
+`;
+
+/**
+ * The "default" view for the panel: A list of all the processes currently in the graph.
+ *
+ * @param {function} pushToQueryparams A function to update the hash value in the URL to control panel state
+ */
+export const ProcessListWithCounts = memo(function ProcessListWithCounts({
+  pushToQueryParams,
+}: {
+  pushToQueryParams: (queryStringKeyValuePair: CrumbInfo) => unknown;
+}) {
+  interface ProcessTableView {
+    name?: string;
+    timestamp?: Date;
+    event: SafeResolverEvent;
+  }
+
+  const dispatch = useResolverDispatch();
+  const { timestamp } = useContext(SideEffectContext);
+  const isProcessTerminated = useSelector(selectors.isProcessTerminated);
+  const handleBringIntoViewClick = useCallback(
+    (processTableViewItem) => {
+      dispatch({
+        type: 'userBroughtProcessIntoView',
+        payload: {
+          time: timestamp(),
+          process: processTableViewItem.event,
+        },
+      });
+      pushToQueryParams({ crumbId: event.entityId(processTableViewItem.event), crumbEvent: '' });
+    },
+    [dispatch, timestamp, pushToQueryParams]
+  );
+
+  const columns = useMemo<Array<EuiBasicTableColumn<ProcessTableView>>>(
+    () => [
+      {
+        field: 'name',
+        name: i18n.translate(
+          'xpack.securitySolution.endpoint.resolver.panel.table.row.processNameTitle',
+          {
+            defaultMessage: 'Process Name',
+          }
+        ),
+        sortable: true,
+        truncateText: true,
+        render(name: string, item: ProcessTableView) {
+          const entityID = event.entityIDSafeVersion(item.event);
+          const isTerminated = entityID === undefined ? false : isProcessTerminated(entityID);
```

Comment:
handle the case where `entityID` is missing. In the near term we should stop passing around events and pass around the entity ID instead.